### PR TITLE
add raspa2

### PR DESCRIPTION
**Pull request by ltalirz**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ltalirz** [comment]
main developer has been pinged in https://github.com/iRASPA/RASPA2/issues/14
will add him as a maintainer of the feedstock in case he is interested (but PR can be merged already)

**Comment by ltalirz** [comment]
One thing that is not quite clear to me: why was there a win build when that should be skipped?
image

Do I need to rerender , despite the win build being skipped from the beginning?

**Comment by jan-janssen** [comment]
> One thing that is not quite clear to me: why was there a win build when I'm skipping this in the meta.yaml?
> image
> 
> Do I need to rerender , despite the win build being skipped from the beginning?

The windows build is only there in the staged - recipes so users do not have to rerender when initially submitting their recipe.

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

**Comment by ltalirz** [comment]
Thanks for the hint, and thanks for asking for review
From the [guidelines](https://github.com/conda-forge/staged-recipes#12-my-pull-request-passes-all-checks-but-hasnt-received-any-attention-how-do-i-call-attention-to-my-pr--what-is-the-customary-amount-of-time-to-wait), it wasn't clear to me whether I *should* ask for review or whether I should just wait for some time

File: recipes/raspa2/build.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+mkdir m4
+aclocal
+autoreconf -i
+automake --add-missing
+autoconf
+./configure --prefix=${PREFIX}
+make
+make install
```
Comment (ocefpaf) [inline recipes/raspa2/build.sh:10]: Is there a `make check` we can run here?

Comment (ltalirz) [reply recipes/raspa2/build.sh:10]: thanks for checking @ocefpaf - there is, but it currently doesn't do anything
```
$ make check
Making check in src
make[1]: Nothing to be done for `check'.
Making check in forcefield
Making check in GenericMOFs
make[2]: Nothing to be done for `check'.
Making check in GenericZeolites
make[2]: Nothing to be done for `check'.
Making check in CrystalGenerator
make[2]: Nothing to be done for `check'.
Making check in Dubbeldam2007FlexibleIRMOF-1
make[2]: Nothing to be done for `check'.
Making check in Dubbeldam2007FlexibleIRMOF-10
make[2]: Nothing to be done for `check'.
Making check in Dubbeldam2007FlexibleIRMOF-16
make[2]: Nothing to be done for `check'.
Making check in GarciaPerez2006
make[2]: Nothing to be done for `check'.
Making check in CoreShellSchroderSauer
make[2]: Nothing to be done for `check'.
Making check in CoreShellSastreGale2005
make[2]: Nothing to be done for `check'.
Making check in CoreShellCatlow
make[2]: Nothing to be done for `check'.
Making check in RigidIonKramer1991
make[2]: Nothing to be done for `check'.
Making check in Pascual2004
make[2]: Nothing to be done for `check'.
Making check in AmirjalayerSchmid
make[2]: Nothing to be done for `check'.
Making check in Nicholas
make[2]: Nothing to be done for `check'.
Making check in Dubbeldam2012MIL-100
make[2]: Nothing to be done for `check'.
Making check in Dubbeldam2012MIL-101
make[2]: Nothing to be done for `check'.
Making check in Dzubak2012-IRMOF-1
make[2]: Nothing to be done for `check'.
Making check in Dzubak2012-MgMOF-74
make[2]: Nothing to be done for `check'.
Making check in Dzubak2012-MgMOF-74-extended
make[2]: Nothing to be done for `check'.
Making check in Dzubak2012-ZnMOF-74
make[2]: Nothing to be done for `check'.
Making check in MgMOF-74-Yazaydin
make[2]: Nothing to be done for `check'.
Making check in CastilloVlugtCalero2009
make[2]: Nothing to be done for `check'.
Making check in TraPPE
make[2]: Nothing to be done for `check'.
make[2]: Nothing to be done for `check-am'.
Making check in molecules
Making check in TraPPE
make[2]: Nothing to be done for `check'.
Making check in TraPPE-UA
make[2]: Nothing to be done for `check'.
Making check in TraPPE-EH
make[2]: Nothing to be done for `check'.
Making check in CastilloVlugtCalero2009
make[2]: Nothing to be done for `check'.
Making check in EPM2
make[2]: Nothing to be done for `check'.
Making check in Generic
make[2]: Nothing to be done for `check'.
make[2]: Nothing to be done for `check-am'.
Making check in framework
Making check in Dubbeldam2007FlexibleIRMOF-1
make[2]: Nothing to be done for `check'.
Making check in Dubbeldam2007FlexibleIRMOF-10
make[2]: Nothing to be done for `check'.
Making check in Dubbeldam2007FlexibleIRMOF-16
make[2]: Nothing to be done for `check'.
Making check in CoreShellCatlow
make[2]: Nothing to be done for `check'.
Making check in CoreShellSchroderSauer
make[2]: Nothing to be done for `check'.
Making check in CoreShellSastreGale2005
make[2]: Nothing to be done for `check'.
Making check in Demontis
make[2]: Nothing to be done for `check'.
Making check in DemontisModified
make[2]: Nothing to be done for `check'.
Making check in Nicholas
make[2]: Nothing to be done for `check'.
Making check in NicholasModified
make[2]: Nothing to be done for `check'.
Making check in AmirjalayerSchmid
make[2]: Nothing to be done for `check'.
make[2]: Nothing to be done for `check-am'.
Making check in structures
Making check in zeolites
Making check in cif
make[3]: Nothing to be done for `check'.
Making check in block
make[3]: Nothing to be done for `check'.
Making check in ions
make[3]: Nothing to be done for `check'.
make[3]: Nothing to be done for `check-am'.
Making check in mofs
Making check in cif
make[3]: Nothing to be done for `check'.
Making check in block
make[3]: Nothing to be done for `check'.
Making check in ions
make[3]: Nothing to be done for `check'.
make[3]: Nothing to be done for `check-am'.
Making check in minerals
Making check in cif
make[3]: Nothing to be done for `check'.
make[3]: Nothing to be done for `check-am'.
Making check in ceramics
Making check in cif
make[3]: Nothing to be done for `check'.
make[3]: Nothing to be done for `check-am'.
make[2]: Nothing to be done for `check-am'.
make[1]: Nothing to be done for `check-am'.
```

I did open an issue concerning a test suite to run https://github.com/iRASPA/RASPA2/issues/12, I suggest that I will include some tests once this has been resolved.
(so far, the recipe just runs `simulate -h`)

Comment (ocefpaf) [reply recipes/raspa2/build.sh:10]: That is a good strategy to make all test pass!

**Pull request merged**